.github/workflows/cluster_endtoend_topo_connection_cache.yml

```diff
@@ -0,0 +1,74 @@
+# DO NOT MODIFY: THIS FILE IS GENERATED USING "make generate_ci_workflows"
+
+name: Cluster (topo_connection_cache)
+on: [push]
```

Comment:
You should add `pull_request` to the list of targets as well. Otherwise, the build will never start on a "regular" commit that lives in a Pull Request. The target `push` only works when a commit is pushed on any branch of `vitessio/vitess` (no its forks).